doc/releases/changelog-dev.md

```diff
@@ -77,6 +77,10 @@
 * The qchem module has dedicated functions for calling `pyscf` and `openfermion` backends.
   [(#5553)](https://github.com/PennyLaneAI/pennylane/pull/5553)
 
+* `qml.from_qasm` now supports the ability to convert mid-circuit measurements from `QASM2` code and it can now also take an
```

Comment:
```suggestion
* `qml.from_qasm` now supports the ability to convert mid-circuit measurements from `OpenQASM 2` code, and it can now also take an
```
```suggestion
* `qml.from_qasm` now supports the ability to convert mid-circuit measurements from `QASM2` code and it can now also take an
```